ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -627,6 +627,30 @@
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if router_bgp.address_family_ipv4_path_selection is arista.avd.defined %}
```

Comment:
```suggestion
{%     if router_bgp.address_family_ipv4_path_selection is arista.avd.defined %}

#### Router BGP IPv4 Path-Selection Address Family
{%         if router_bgp.address_family_ipv4_path_selection.neighbors is arista.avd.defined %}
```
And also remember to fix jinja indentation and add an `endif` more after the whole section :)